src/runtime/dom-particle-base.ts

```diff
@@ -220,7 +220,7 @@ export class DomParticleBase extends Particle {
    */
   async appendRawDataToHandle(handleName: string, rawDataArray): Promise<void> {
     const handle = this.handles.get(handleName);
-    if (handle && handle.entityClass) {
+    if (handle && handle['entityClass']) {
```

Comment:
I think checking for `handle && handle instanceof PreMutationEntityHandle` would be a bit clearer